src/ports/postgres/modules/graph/graph_utils.py_in

```diff
@@ -109,6 +110,85 @@ def validate_graph_coding(vertex_table, vertex_id, edge_table, edge_params,
 
     return None
 
+def validate_params_for_centrality_measures(schema_madlib, func_name,
```

Comment:
This function name is a bit confusing since it isn't used by the centrality measures functions from `measures.py_in`

Reply:
+1